File: src/ConstructTheRectangle.java
public class ConstructTheRectangle {
    public int[] constructRectangle(int area) {
        int width = (int) Math.sqrt(area);

        while (width > 0) {
            if (area % width == 0) {
                return new int[] {area / width , width};
            }
            width--;
        }

        return new int[] {area , 1};
    }
}

File: python/construct_the_rectangle.py
from math import sqrt
from typing import List


class Solution:
    def constructRectangle(self, area: int) -> List[int]:
        width = int(sqrt(area))

        while width > 0:
            if area % width == 0:
                return [area // width, width]
            else:
                width -= 1

        return [area, 1]
